### ISSUE TITLE
diybio Group Issue

### ISSUE THREAD

**Issue by joshuamcginnis** [DIYbiosphere/sphere#299]
I made a post last week on the [diybio google group](https://groups.google.com/g/diybio/) but it looks like it never made it past the moderation queue. 

I tried reaching out to @100ideas and the other group owners per the about page, but then I got an email from google saying:

> This group does not accept email to owners or managers. To send a message to the owner, visit the following page: https://groups.google.com/d/contactowner/diybio` 

The email just links back to the about page with the same `Contact owners and managers` link that apparently isn't supported.

Basically, there is no way to contact the owners of the group and inquire as to if/why my post last week never made it live and any attempts to contact the owners puts you into a recursive loop.

Maybe it's me! Is there some other way I should inquire as to why my posts aren't going live? I thought only posts from new members went into the moderation queue. I've been a member for a year+ at this point.

Thanks! Josh

**Comment by jasonbobe** [comment]
Hi Josh - I just approved the message, sorry to be asleep at the wheel!

Jason

**Comment by joshuamcginnis** [comment]
Thank you kindly! 🙏
